app/api/v2/models/parcel.py

```diff
@@ -0,0 +1,33 @@
+"""Docstring This class for our parcel model"""
+from ....db_config import init_db
+import uuid
+
+
+class ParcelModel:
+    """"This is the class parcel model for testing model"""
+
+    def __init__(self):
+        """initialzation for our data"""
+        self.con = init_db()
+
+    def add_parcel(self, data, user_id):
+        """This method adds parcels to our parcels"""
+        parcel_type = data['parcel_type']
+        dest = data['Dest']
+        dest_from = data['Dest_from']
```

Comment:
Call the starting point `pickup_location`? And the dropping point `drop_off_location`?